site/docs/latest/admin/upgrade.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Upgrade
+---
+
+> If you have questions about upgrades (or need help), please feel free to reach out to us by [mailing list](/community/mailing-lists) or [Slack Channel](/community/slack).
+
+## Overview
+
+Consider the below guidelines in preparation for the upgrade.
+
+- Always back up all your configuration files before upgrading.
+- Read through the documentation and draft an upgrade plan that matches your specific requirements and environment before starting the upgrade process.
+    Put differently, don't start working through this guide on a live cluster. Read guide entirely, make a plan, then execute the plan.
+- Pay careful consideration to the order in which components are upgraded. In general, you need to upgrade `bookies` first and then upgrade your clients.
```

Comment:
"Autorecovery" and "bookies" don't need to be in backticks on this page (because the terms aren't code)